dojo/api_v2/serializers.py

```diff
@@ -1058,8 +1058,13 @@ class FindingCreateSerializer(TaggitSerializer, serializers.ModelSerializer):
 
     class Meta:
         model = Finding
-        exclude = ['images']
+        fields = '__all__'
         extra_kwargs = {
+            'mitigation': {'required': True},
+            'impact': {'required': True},
```

Comment:
I don't believe these fields are required anywhere in the app anymore, but they were when the issue was created.